libraries/AP_InertialSensor/AP_InertialSensor_Invensense.cpp

```diff
@@ -46,6 +47,10 @@ extern const AP_HAL::HAL& hal;
 #define debug(fmt, args ...)  do {printf("MPU: " fmt "\n", ## args); } while(0)
 #endif
 
+#ifndef HAL_ENABLE_FAST_FIFO_RESET_ICM20602
+#define HAL_ENABLE_FAST_FIFO_RESET_ICM20602 false
```

Comment:
I think this should be 1/0 or TRUE/FALSE - I don't think we use lower case anywhere

Reply:
Not seeing where this is actually used?